pkg/beacon/analyzer.go

```diff
@@ -323,31 +327,47 @@ func getTsHistogramScore(min int64, max int64, tsList []int64) ([]int64, []int,
 		cv = 1.0
 	}
 
-	// calculate first potential score
-	// histograms with bigger flat sections will score higher, up to 4 flat sections
-	// this will score well for graphs that have flat sections with a big distance between them,
-	// i.e. a beacon that alternates between 1 and 5 connections per hour
-	// This score will only be calculated if the number of total bars on the histogram fills at
-	// least half the day (>11), otherwise this could lead to a false positive and not fill the
+	cvScore := math.Ceil((1.0-float64(cv))*1000) / 1000
+	if cvScore > 1.0 {
+		cvScore = 1.0
+	}
+
+	// Calculate second potential score
+	// this will score well for graphs that have 2-3 flat sections in their connection histogram,
+	// or a bimodal freqCount histogram.
+	// Example - a beacon that alternates between 1 and 5 connections per hour
+	// This score will only be calculated if the number of total bars on the histogram is at
+	// least 10, otherwise this could lead to a false positive and not fill the
 	// original intent of this scoring.
-	score1 := 0.0
+	bimodalFit := float64(0)
+
+	if totalBars > 10 {
```

Comment:
This just checks if there are more than 10 non zero hours in the frequency list. If not, the score stays at 0.

Since the max denominator for the score is 23 (24 total hours in the dataset - 1 outlier) I believe our minimum step size for the score should be 1/23. There is an example of the score (13/14) in example 3 above.

Is there anything I'm missing that would pin the increment to 10%?

Reply:
I see what you were saying. My intention with this wasn't a step size, but yes, the maximum step size would be 10%. The original version of the binomial version of the histogram score only applied if there were [more than 11 bars](https://github.com/activecm/rita/blob/9819b72d60ac3e12d9ccee478decdea3809d06c5/pkg/beacon/analyzer.go#L335) . This was to keep cases like this from scoring high:
<img width="1302" alt="image" src="https://user-images.githubusercontent.com/6879542/234081622-6d4dfa4f-6ee0-447b-8cd5-9b96f9509cc5.png">
I lowered that threshold a bit in this PR, but the intention remains the same. I don't think that few of bars is enough to draw conclusions about consistency. 
I do see what you are saying about the step size. Is there a reason for it to be detrimental for the step size to be capped at 10%?

Reply:
> The reasoning is making it just shy of half the day.
That's what I was missing. Please update the comment to include the step size and this note about the cutoff being a rule of thumb based on it being roughly half the day.